src/identities.rs

```diff
@@ -280,8 +276,8 @@ impl UserIdentity {
     /// True if we verified this identity (with any own identity, at any
     /// point).
     ///
-    /// To pass this latch back to false, one must call
-    /// [`UserIdentity::withdraw_verification()`].
+    /// To set this latch back to false, call
+    /// {@link #withdrawVerification}.
```

Comment:
```suggestion
    /// {@link withdrawVerification}.
```

it doesn't seem to like the "#"